### DISCUSSION TITLE
Wrong results

### DISCUSSION THREAD

**Discussion by unfairDude** [blackhatethicalhacking/TerminatorZ#10] [Q&A]
I performed a full scan on vulnweb.com
I got 0 vulnerabilities, how is that possible?

also i get DBG errors at beginning!
[Screenshot_2023-04-24-02-12-22-659_com.sonelli.juicessh.jpg](https://user-images.githubusercontent.com/102629559/234130256-7c0c09a6-b4ea-425d-a298-bed0745a1190.jpg)

**Comment by blackhatethicalhacking** [comment]
These errors means that these URLs no longer are available, and HTTPX is correctly stating that. they are normal errors, say you harvest 1000 urls, passively, from these 1000 urls - a lot are no longer available, and these DBG tell you this.
